app/src/main/kotlin/com/google/samples/apps/nowinandroid/NiaApplication.kt

```diff
@@ -34,6 +36,14 @@ class NiaApplication : Application(), ImageLoaderFactory {
 
     override fun onCreate() {
         super.onCreate()
+
+        // Kill NiA if there are main thread policy violations.
+        if (BuildConfig.DEBUG) {
+            StrictMode.setThreadPolicy(
+                StrictMode.ThreadPolicy.Builder().detectAll().penaltyDeath().build(),
```

Comment:
Add `.penaltyLog()` would make life easier because it indicates which line of code violates the StrictMode.

Reply:
Good call. Added logging.